tests/regression/36-apron/86-branched-thread-creation.c

```diff
@@ -40,7 +40,7 @@ int main(void) {
 
   if(!mt) {
     pthread_mutex_lock(&mutex);
-    __goblint_check(g==h); //MAYFAIL
+    __goblint_check(g==h); //MAY FAIL
```

Comment:
Even though, this was not the content of this PR, this annotation should actually be a `FAIL` annotation since the check should always fail on all paths.